src/mranderson/move.clj

```diff
@@ -292,12 +292,18 @@
   "Replaces all occurrences of the old name with the new name in
   all Clojure source files found in dirs."
   [old-sym new-sym extension dirs watermark]
-  (->> (clojure-source-files dirs extension)
-       (pmap-runner (fn [file]
-                      (->> (str file)
-                           util/file->extension
-                           (update-file file replace-ns-symbol old-sym new-sym watermark extension))))
-       (doall)))
+  (let [files (clojure-source-files dirs extension)
+        dupes (->> (map str files)
+                   frequencies
+                   (filterv #(> (second %) 1)))]
+    (if (seq dupes)
+      (throw (ex-info "internal error: found unexpected duplicates in files" {:dupes dupes}))
```

Comment:
Looks like this path could get coverage?

Reply:
Only if we have an unintentional programming error.

Reply:
I mean, how do we know that the `frequencies` code works?

You could exercise that by intentionally invoking this defn with bad input.

Reply:
Oh right. A unit test. 🙂 Yeah can do.
This was the first change I put in and it get triggered before other fixes were put in.